### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chStaiger**
- raw link to logo
- link to gui

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ iBridges is a library for scientific programmers who are working with data in iR
 - Small number of dependencies (`python-irodsclient` and `tqdm`)
 - Safe default options when working with your data.
 
+**iBridges** also has a braphicl user interface, which you can install [here](https://github.com/chStaiger/iBridges-Gui).
```
Comment (trel) [inline README.md:23]:
```suggestion
**iBridges** also has a graphical user interface, which you can install [here](https://github.com/chStaiger/iBridges-Gui).
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# iBridges <img src="docs/logo.png" width="150" align="right">
+# iBridges <img src="https://github.com/UtrechtUniversity/iBridges/blob/readme/docs/logo.png?raw=true" width="150" align="right">
```
Comment (trel) [inline README.md:1]: you linked to this branch (readme)

Comment (chStaiger) [reply README.md:1]: Ah, thanks for spotting that!!!

Comment (chStaiger) [inline README.md:1]:
```suggestion
# iBridges <img src="https://github.com/UtrechtUniversity/iBridges/blob/main/docs/logo.png?raw=true" width="150" align="right">
```

File: docs/requirements.txt
[the remark this answers is not in the document]
Comment (chStaiger) [reply docs/requirements.txt:1]: Ah yes. 

Currently I am trying to solve our workflow for the RTD rendering:

```
Theme error:
An error happened in rendering the page api/full_reference.
Reason: UndefinedError("'style' is undefined")
make: *** [Makefile:20: html] Error 2
Error: Process completed with exit code 2.
```

File: docs/requirements.txt [hunk 3]
```diff
@@ -1,6 +1,5 @@
...
-sphinx_inline_tabs
-sphinx-rtd-theme
-sphinxcontrib-napoleon
-sphinx-autodoc-typehints
-sphinx_copybutton
+sphinx<8.0.0
+sphinx-autodoc-typehints==2.4.1
+sphinx-copybutton==0.5.2
+sphinx-rtd-theme==2.0.0
+sphinx_inline_tabs==2023.4.21
```
Comment (chStaiger) [inline docs/requirements.txt:5]:
```suggestion
sphinx<9.0.0
sphinx_inline_tabs
sphinx-rtd-theme
sphinxcontrib-napoleon
sphinx-autodoc-typehints
sphinx_copybutton
```

**Pull request merged**